modules/preview-react/form-field/lib/hooks/useFormFieldModel.tsx

```diff
@@ -16,13 +16,12 @@ export const useFormFieldModel = createModelHook({
      * - `FormField.Hint` will set `id` to `hint-${id}`
      *
      * If a value is not provided, a unique id will be automatically created by `useUniqueId()`.
-     * @default `useUniqueId()`
+     * @default {useUniqueId}
      */
     id: '',
     /**
      * Optional flag to denote if this field is required. When true the `FormField.Input` will have
      * `required` set to true, and an asterisk will be appended to the `FormField.Label`.
-     * @default false
```

Comment:
I'm not sure why this was removed.

Reply:
These default tags are not needed. They are inside a block called `defaultConfig`, meaning the value _is_ the default. Also the model parser understands this context.

I'd rather not keep default values in sync everywhere they can be inferred.